utils/k6_prepare_tracer.py

```diff
@@ -0,0 +1,253 @@
+                        "sender_balance_after": f"{self.web3_client.get_balance(sender_account)}",
+                        "sender_nonce": f"{self.web3_client.get_nonce(sender_account)}",
+                        "recipient_balance_before": f"{recipient_balance_before}",
+                        "recipient_balance_after": f"{self.web3_client.get_balance(recipient_account)}",
+                        "amount": transfer_amount
+                    })
+            except Exception as e:
+                print(f"Error in neon_transfer_data: {e}") 
+
+    def erc20_wrapped_transfer_data(self, transfers_number):
+        account_owner = self.get_account(balance=1000)
+        
+        erc20_wrapper = ERC20Wrapper(
+            self.web3_client,
+            self.faucet,
+            "Test Token",
+            "TTW",
+            self.sol_client,
+            solana_account=Keypair(),
+            account=account_owner,
+            mintable=True,
+        )
+        print("ERC20 wrapped contract deployment...")
+        erc20_wrapper.deploy_wrapper(True)
+        erc20_wrapper.mint_tokens(account_owner, account_owner.address, 18446744073709551615)
+        
+        print("ERC20 wrapped transfers...")
+        for i in range(transfers_number):
+            try:
+                transfer_amount = random.randint(1, 5)
+                account_receiver = self.get_account(balance=0)
+                sender_balance_before = erc20_wrapper.get_balance(erc20_wrapper.account)
+                recipient_balance_before = erc20_wrapper.get_balance(account_receiver)
+                
+                receipt = erc20_wrapper.transfer(erc20_wrapper.account, account_receiver, transfer_amount)
+                
+                print(f"ERC20 wrapped transfer {i}, receipt status: ", receipt["status"])
+                self.historical_data["erc20spl_transfers"].append({
+                        "blockHash": receipt["blockHash"].hex(),
+                        "blockNumber": hex(receipt["blockNumber"]),
+                        "erc20spl_contract_address": erc20_wrapper.contract_address,
+                        "sender": erc20_wrapper.account.address,
+                        "recipient": account_receiver.address,
+                        "sender_balance_before": f"{sender_balance_before}",
+                        "sender_balance_after": f"{erc20_wrapper.get_balance(erc20_wrapper.account)}",
+                        "sender_nonce": f"{self.web3_client.get_nonce(erc20_wrapper.account)}",
+                        "recipient_balance_before": f"{recipient_balance_before}",
+                        "recipient_balance_after": f"{erc20_wrapper.get_balance(account_receiver)}",
+                        "amount": transfer_amount
+                    })
+            except Exception as e:
+                print(f"Error in erc20_transfer_data: {e}") 
+
+    def storage_contract_data(self, calls_number):
```

Comment:
it will be good to add to one of the solana calls "value" field and check balances not only for "send neon" scenario

Reply:
```solana calls "value" field``` what do you mean?

Reply:
Send tokens to contract address when you call this contract

You have cases with sending tokens to some user
You have cases with calling functions of contracts
But there are no cases when you change contract balance

Reply:
Calling a contract balance or an account one works same inside our tracer: we just take needed value from db, I can add such case just for test data variety I guess